src/qml/qmlplayermanagerproxy.cpp

```diff
@@ -66,19 +66,22 @@ QmlPlayerManagerProxy* QmlPlayerManagerProxy::create(QQmlEngine* pQmlEngine, QJS
     // that shows the replacement for `qmlRegisterSingletonInstance()` when
     // using `QML_SINGLETON`.
     // https://doc.qt.io/qt-6/qqmlengine.html#QML_SINGLETON
+    // FIXME: this pattern isn't compatible with multiple engines sparsed in
```

Comment:
@Holzhaus do you have any idea how I could make your implementation work with a multiple engine/thread? the doc seems to say I should use `qmlRegisterSingletonInstance` explicitly, which I believe was the case before your refactor.